_common/KeymanSiteEnvironment.php

```diff
@@ -0,0 +1,22 @@
+<?php
+  declare(strict_types=1);
+
+  namespace Keyman\Site\Common;
+
+  class KeymanSiteEnvironment {
+
+    protected static function _Init($class) {
+      $env = getenv();
+      $props = get_class_vars($class);
+      foreach($props as $name => $value) {
+        if (isset($env[$name])) {
+          self::$instance->$name = $env[$name];
+        }
+      }
+    }
+
+    public static function Debug($class) {
+      $props = get_class_vars($class);
+      var_dump($props);
+    }
+  }
```

Comment:
Looking ahead to refactoring

https://github.com/keymanapp/api.keyman.com/blob/9bb62cc86edd9b25fc765bf78c18120e7aed1002/tests/TestDBBuild.inc.php#L49-L51

and

https://github.com/keymanapp/api.keyman.com/blob/9bb62cc86edd9b25fc765bf78c18120e7aed1002/tests/TestUtils.inc.php#L18-L22

Will we functions to return a boolean if the env var is `''`?

Reply:
> 1. unset a property

I'd prefer to rewrite how that particular function works. It's suboptimal at present.

> 2. return a boolean if the env var is `''`?

No, if the env var is unsset, then the property value should be `NULL` (testable with is_null, isset, isempty etc).

Reply:
> No, if the env var is unsset, then the property value should be NULL ...

I meant for 2. to check when vars are initialized to `''`

https://github.com/keymanapp/help.keyman.com/blob/9169d57566ca64026b4421b2fc4a80cc7b7f38b1/_includes/HelpSiteEnvironment.php#L9-L12

Reply:
If it is needed?